trailmet/algorithms/prune/chipnet.py

```diff
@@ -195,35 +200,37 @@ def prune(self, model, dataloaders, **kwargs):
                 pruning_threshold.append(threshold)
                 remaining_after_pruning.append(remaining)
                 problems.append(problem)
-                
+
                 # 
-                beta=min(6., beta+(0.1/self.b_inc))
-                gamma=min(256, gamma*(2**(1./self.g_inc)))
+                beta = min(6., beta + (0.1 / self.b_inc))
+                gamma = min(256, gamma * (2**(1. / self.g_inc)))
                 self.set_beta_gamma(beta, gamma)
-                print("Changed beta to", beta, "changed gamma to", gamma)     
-                
-                if acc>best_acc:
+                print("Changed beta to", beta, "changed gamma to", gamma)
+
+                if acc > best_acc:
                     print("**Saving checkpoint**")
-                    best_acc=acc
+                    best_acc = acc
                     torch.save({
-                        "epoch" : epoch+1,
-                        "beta" : beta,
-                        "gamma" : gamma,
-                        "prune_threshold":threshold,
-                        "state_dict" : model.state_dict(),
-                        "accuracy" : acc,
+                        "epoch": epoch+1,
+                        "beta": beta,
+                        "gamma": gamma,
+                        "prune_threshold": threshold,
+                        "state_dict": model.state_dict(),
+                        "accuracy": acc,
                     }, f"checkpoints/{self.log_name}.pth")
 
                 df_data=np.array([remaining_before_pruning, remaining_after_pruning, valid_accuracy, pruning_accuracy, pruning_threshold, problems]).T
                 df = pd.DataFrame(df_data,columns = ['Remaining before pruning', 'Remaining after pruning', 'Valid accuracy', 'Pruning accuracy', 'Pruning threshold', 'problems'])
                 df.to_csv(f"logs/{self.log_name}.csv")
 
     def steepness_update_function(self, step):
+        """returns function to update steepness in budget loss of chipnet"""
         def update():
             self.steepness = min(60, self.steepness+step)
```

Comment:
add comment why the default is set to 60.

Reply:
done